regrippy/plugins/auditpol.py

```diff
@@ -0,0 +1,157 @@
+from regrippy import BasePlugin, PluginResult, mactime
+from struct import unpack
+from collections import OrderedDict
+
+
+POL = OrderedDict.fromkeys([
+    'System',
+    'Logon/Logoff',
+    'Object Access',
+    'Privilege Use',
+    'Detailed Tracking',
+    'Policy Change',
+    'Account Management',
+    'DS Access',
+    'Account Logon'
+])
+
+POL['System'] = [
+    'Security State Change',
+    'Security System Extension',
+    'System Integrity',
+    'IPsec Driver',
+    'Other System Events'
+]
+
+POL['Logon/Logoff'] = [
+    'Logon',
+    'Logoff',
+    'Account Lockout',
+    'IPsec Main Mode',
+    'Special Logon',
+    'IPsec Quick Mode',
+    'IPsec Extended Mode',
+    'Other Logon/Logoff Events',
+    'Network Policy Server',
+    'User/Device Claims',
+    'Group Membership'
+]
+
+POL['Object Access'] = [
+    'File System',
+    'Registry',
+    'Kernel Object',
+    'SAM',
+    'Other Object Access Events',
+    'Certification Services',
+    'Application Generated',
+    'Handle Manipulation',
+    'File Share',
+    'Filtering Platform Packet Drop',
+    'Filtering Platform Connection',
+    'Detailed File Share',
+    'Removable Storage',
+    'Central Policy Staging'
+]
+
+POL['Privilege Use'] = [
+    'Sensitive Privilege Use',
+    'Other Privilege Use Events',
+    'Non Sensitive Privilege Use'
+]
+
+POL['Detailed Tracking'] = [
+    'Process Creation',
+    'Process Termination',
+    'DPAPI Activity',
+    'RPC Events',
+    'Plug and Play Events',
+    'Token Right Adjusted Events'
+]
+
+POL['Policy Change'] = [
+    'Audit Policy Change',
+    'Authentication Policy Change',
+    'Authorization Policy Change',
+    'MPSSVC Rule-Level Policy Change',
+    'Filtering Platform Policy Change',
+    'Other Policy Change Events'
+]
+
+POL['Account Management'] = [
+    'User Account Management',
+    'Computer Account Management',
+    'Security Group Management',
+    'Distribution Group Management',
+    'Application Group Management',
+    'Other Account Management Events'
+]
+
+POL['DS Access'] = [
+    'Directory Service Access',
+    'Directory Service Changes',
+    'Directory Service Replication',
+    'Detailed Directory Service Replication'
+]
+
+
+POL['Account Logon'] = [
+    'Credential Validation',
+    'Kerberos Authentication Service',
+    'Other Account Logon Events',
+    'Kerberos Service Ticket Operations'
+]
+
+
+LOGTYPES = {0: 'No Auditing', 1: 'Success', 2: 'Failure', 3: 'Success/Failure'}
+
+class Plugin(BasePlugin):
+    """Get the Adv audit policy"""
```

Comment:
Could you make this more descriptive? Such as "Get the advanced security audit policy settings"

Reply:
Done in b72063bf81ca6473285631b746c14ecb79d2e50f